### PR TITLE
Docker healthcheck

### PR DESCRIPTION
**Pull request by vincentlepot**
Add healthchecks on both php and nginx prod images

### CODE REVIEW

File: docker/nginx/Dockerfile [hunk 1]
```diff
@@ -11,3 +17,6 @@ COPY web/robots.txt /srv/hermod/web/robots.txt
 COPY web/favicon.ico /srv/hermod/web/favicon.ico
 
 CMD ["/run.sh"]
+
+HEALTHCHECK --interval=30s --timeout=3s \
```
Comment (Netmisa) [inline docker/nginx/Dockerfile]: You can change `--interval=30s` to `--interval=5s`
`30s` is too long

File: docker/php/Dockerfile [hunk 2]
```diff
@@ -39,3 +46,9 @@ RUN chown -R www-data:www-data var/cache var/logs var/sessions
 USER www-data
 
 CMD ["/srv/hermod/docker/php/run.sh"]
+
+HEALTHCHECK --interval=30s --timeout=3s \
```
Comment (Netmisa) [inline docker/php/Dockerfile]: (like above)
You can change `--interval=30s` to `--interval=5s`
`30s` is too long

**Comment by Netmisa** [comment]
Work for me

File: docker/php/Dockerfile (hunk 2 above)
Comment (vincentlepot) [reply docker/php/Dockerfile]: Done

File: docker/nginx/Dockerfile (hunk 1 above)
Comment (vincentlepot) [reply docker/nginx/Dockerfile]: Done

**Pull request merged**